Composer/packages/server/src/services/telemetry.ts

```diff
@@ -69,7 +69,7 @@ if (instrumentationKey) {
 
       // remove PII
       for (const property of piiProperties) {
-        if (data.baseData.properties[property]) {
+        if (typeof data.baseData.properties[property] !== 'undefined') {
```

Comment:
general comment: you can just compare with ```undefined``` itself instead of comparing the typeof with a string